CONTRIBUTING.md

```diff
@@ -112,7 +112,11 @@ of the settings.
 
 If you are on Windows, you need to install **make** for the above process to complete.
 
+<<<<<<< HEAD
 > **Note**
+=======
+> **Note**:
+>>>>>>> 1c24fa0 (fixing issues)
```

Comment:
you have some merge artifacts

Reply:
i hate this pr by now - i already rebased it so often - because of merge conflicts :( - and always is there something more

Reply:
deleted my fix commit, and reapplied markdownlint and created one new commit for each rule.